flight/Modules/Airspeed/airspeed.c

```diff
@@ -104,10 +104,10 @@ static int8_t airspeedADCPin=-1;
 // Private functions
 static void airspeedTask(void *parameters);
 void baro_airspeedGet(BaroAirspeedData *baroAirspeedData, uint32_t *lastSysTime, uint8_t airspeedSensorType, int8_t airspeedADCPin);
-static void AirspeedSettingsUpdatedCb(UAVObjEvent * ev);
+static void AirspeedSettingsUpdatedCb(UAVObjEvent * ev, void *ctx, void *obj, int len);
```

Comment:
The context record is so you can call the same callback multiple times for different "purposes".

This is important if we are ever going to have multiple instances of a module running.

For instance, if, say, CameraStab takes callbacks, you can (eventually) instantiate the CameraStab subsystem twice, have it register twice, and it uses the ctx variable for its state (instead of just peeking at globals which doesn't allow you to have more than one, at least not without a lot of code to handle multiple instantiations).  Or telemetry or whatever.  

Another instance of how I use it is the UAVObjCbSetFlag / UAVObjCbCopyData-- they're code that implements a very common pattern of setting a flag or copying a uav object to a given place when it changes.  The context pointer is the place where the module keeps the flag or where to copy the data to.